web/pdf_page_view.js

```diff
@@ -142,6 +142,25 @@ class PDFPageView {
     }
   }
 
+  /**
+   * @private
+   */
+  async _renderAnnotationLayer() {
+    let error = null;
+    try {
+      await this.annotationLayer.render(this.viewport, "display");
+      error = false;
```

Comment:
I think having `error` either be `null` or an actual value is more consistent with for example the API methods, so it looks better to me if this line is removed.